iteration/repeat_test.go

```diff
@@ -3,39 +3,36 @@ package iteration
 import (
 	"fmt"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestRepeat(t *testing.T) {
 
-	assertCorrectMessage := func(t testing.TB, repeated, expected string) {
-		t.Helper()
-		if repeated != expected {
-			t.Errorf("expected %q but got %q", expected, repeated)
-		}
-	}
+	assert := require.New(t)
 
-	t.Run("Repeat a character, 10 times", func(t *testing.T) {
-		repeated := Repeat("a", 10)
-		expected := "aaaaaaaaaa"
-		assertCorrectMessage(t, repeated, expected)
+	t.Run("Repeat repeats a character, 10 times", func(t *testing.T) {
+		want := Repeat("a", 10)
+		got := "aaaaaaaaaa"
+		assert.Equal(want, got)
```

Comment:
Now that you are also passing the `expectedError` like [here](https://github.com/rjar2020/learn-go-with-tests/pull/10/files#diff-8e865e7a462799f25b246b6030575b9c3b0a3f6e73cc70781c175743559878c8R28). I'd it for the rest of the refactor. What do you think? :smiling_imp:

Reply:
Even though I'm not keen on "One size fits all" approaches, let's agree that when sub-tests we write a clear description so we don't need a redundant message in assertions. When not subtests, let's use assertion's messages. Fix [already committed](https://github.com/rjar2020/learn-go-with-tests/pull/10/commits/fbb50063846eb9d7e24553be23b83f81f0747202) to match this agreement. 

What do you think @JuanCalcagno416 ? Cheers!

Reply:
Perfect then, I even prefer to not pass it at all, to be honest :)